app/services/idv/native_camera_a_b_test.rb

```diff
@@ -0,0 +1,14 @@
+module Idv
+  class NativeCameraABTest < AbTestBucket
```

Comment:
I'm not sure I'd subclass like this, but I do like the idea of fixed, constant buckets for each so we don't have to constantly reallocate and check

like in an intiailizer

```ruby
module AbTests
  NATIVE_CAMERA = AbTestBucket.new(....)
end

# ...

AbTests::NATIVE_CAMERA.bucket(uuid)
```

Reply:
Fair enough. Done in dafee2214

Reply:
scratch that - having too many problems so reverted to inherited class 😞